docs/building.md

```diff
@@ -5,7 +5,25 @@ Ensure you have JDK 8 (or newer), Maven 3.5.4 (or newer) and Git installed
     java -version
     mvn -version
     git --version
-    
+
+**NOTE**
+
+For users of _Maven 3.8.1_ or newer: 
+Please add the following mirror to your local _~/.m2/settings.xml_ to avoid build errors:
```

Comment:
Just a nitpick (to avoid double colon). I'd use maybe something like:
```suggestion
If you use Maven 3.8.1 or newer, please add the following mirror to your local `~/.m2/settings.xml` to avoid build errors:
```